src/MuladdMacro.jl

```diff
@@ -87,23 +90,33 @@ end
 
 Determine whether `ex` is a dot call.
 """
-isdotcall(ex) =
-    (@capture(ex, (f_)(__)) && startswith(string(f), '.')) ||
-    @capture(ex, (_).(__))
+isdotcall(ex) = typeof(ex) <: Expr ? ex.head == :. || string(ex.args[1])[1] == '.' : false
+
+"""
+    iscall(ex, op)
+
+Determine whether `ex` is a call to `op`.
+"""
+function iscall(ex, op)
+    !(typeof(ex) <: Expr) && return false
+    length(ex.args) == 0 && return false
+    _op = ex.args[1]
+    return _op == op || _op == Symbol(:., op)
+end
```

Comment:
This seems a not very detailed check to me but I'm not sure whether it has to be more detailed actually. Usually we just want to catch the cases with `ex.head == :call` and `ex.args[1] == op` or `ex.args[1] == Symbol(:., op)` and with `ex.head == :.`, `ex.args[1] == op` (and `ex.args[2]` a tuple expression). I'm curious whether we might catch unwanted "sums" or "subtractions" in `issum` and `issub` because of this.